common/production_methods/babylon_naval.txt

```diff
@@ -315,29 +320,34 @@ pm_paixhan_guns = {
 	}
 }
 
-pm_80_pound_dahlgren_guns = {
-	texture = "gfx/interface/icons/military_icons/skull.dds"
+pm_6_pounder_hotchkiss_gun = {
+	texture = "gfx/interface/illustrations/icons/military_icons/ironclad.dds"
 	unlocking_technologies = {
-		explosive_armaments
+		naval_artillery
+	}
+	unlocking_production_methods = {
+		pm_ironclads
+		pm_monitors
+		pm_dreadnoughts
+		pm_battleships
 	}
-
 	building_modifiers = {
 		level_scaled = {
-			building_training_rate_add = 5
-			building_employment_soldiers_add = 225
-			building_employment_officers_add = 7
+			building_training_rate_add = 4
+			building_employment_soldiers_add = 150
+			building_employment_officers_add = 5
```

Comment:
Oh I forgot to tell you, we can't increase employment like this, the game gives a standard amount of flotilla per X amount of employees. The admirals get assigned N buildings, depending on capacity, this means that N buildings will produce N * 1.155 flotillas in this case. This X is a constant, so we can't make the employment increase over time. 

We need to fix the amount of employees at the start of the game, and then change the constant if we want it to be more than 1000 per flotilla.

Reply:
How do we change the fixed amount of employees at game start?

Reply:
Or just do any of that in general. I of course wouldn't want to have us be producing excess flotillas

Reply:
> How do we change the fixed amount of employees at game start?

By changing a define max_manpower in 00_combat_unit_types.txt. Or what I mean is, progression through the production_method_groups shouldn't increase total employment at all.

Reply:
I see, so we just remove the extra employment and/or change things around to make it so that it caps at 1k employees